cli/jobs/pipelines/iris-batch-prediction-using-parallel/pipeline.yml

```diff
@@ -10,36 +10,32 @@ tags:
 settings:
   default_compute: azureml:cpu-cluster
 
-inputs:
-  job_data_path:
-    type: mltable
-    path: ./neural-iris-mltable
-    mode: direct
-  score_model:
-    type: uri_folder
-    path: ./iris-model
-    mode: download
-
-outputs:
-  job_out_file:
-    type: uri_file
-    mode: rw_mount
-
 jobs:
   batch_prediction:
     type: parallel
     compute: azureml:cpu-cluster
     inputs:
-      input_data: ${{parent.inputs.job_data_path}}
```

Comment:
why do we remove the binding?

Reply:
Want to reduce some complexity when user start to learn PRS job in pipeline.